packages/dds/tree/src/test/memory/tree.spec.ts

```diff
@@ -33,28 +39,65 @@ class RootNodeSchema extends builder.object("root-item", {
 	child: builder.optional(ChildNodeSchema),
 }) {}
 
-function createLocalSharedTree(id: string): TreeView<typeof RootNodeSchema> {
-	const sharedTree = SharedTree.create(
+const initialState = {
+	child: {
+		propertyOne: 128,
+		propertyTwo: {
+			itemOne: "",
+		},
+		propertyThree: new Map([["numberOne", 1]]),
+	},
+};
+
+// Array has 2 allowable types to make it less efficient for uniform chunks.
+class RootNodeSchemaBasicChunks extends builder.array("root-item-with-basic-chunks", [
+	builder.number,
+	builder.string,
+]) {}
+
+class RootNodeSchemaUniform extends builder.array(
+	"root-item-with-basic-chunks",
+	builder.number,
+) {}
+
+class NestedNodeSchema extends builder.object("wrapped-item", {
```

Comment:
For these tests its important that this is monomorphic. Maybe call it DeepMonomorphic.

Also Schema classes should generally not have "Schema": in their name since that name is also used for the nodes with that schema. If the name is too generic, adding "Node" to the end is probably better than Schema.

Reply:
Updated with suggested name